crates/bevy_render/src/render_resource/buffer_vec.rs

```diff
@@ -3,13 +3,15 @@ use crate::{
     renderer::{RenderDevice, RenderQueue},
 };
 use bevy_core::{cast_slice, Pod};
+use std::ops::{Deref, DerefMut};
 use wgpu::BufferUsages;
 
+/// A user-friendly wrapper around a [`Buffer`] that provides a `Vec`-like
+/// interface for constructing the buffer.
 pub struct BufferVec<T: Pod> {
     values: Vec<T>,
     buffer: Option<Buffer>,
     capacity: usize,
-    item_size: usize,
```

Comment:
I really like removing this: it reduces the risk of terrible bugs. `size_of` is a const_fn, so we should probably actually see tiny perf gains from this too.

Reply:
This seems to be a holdover from (Dynamic)UniformBufferVec, where it is actually needed. But here it was basically just a const value.